news/2025/2025-07-28-osucatch-world-cup-2025-concludes.md

```diff
@@ -0,0 +1,462 @@
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/6567640" style="background-image: url('/wiki/shared/avatars/Realmaas.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-loved" href="https://osu.ppy.sh/users/6567640">Realmaas</a></p>
+
+How did your team approach or prepare for this year's bracket? Did anything change compared to previous years?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/8172283" style="background-image: url('/wiki/shared/avatars/-Story-.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/8172283">-Story-</a></p>
+
+Last year I feel like I performed very well as captain in terms of deciding what to pick and what strategy to use to win matches, but this year I felt more relaxed regarding practice, choosing picks and at the team's level. Our way of practicing is simple: we take maps that we consider difficult and then organize them into a Google Sheet to upload scores and improve on the current type of maps within the year to understand the lastest skillsets from 2025. So, pretty much we changed a bit the strategy and choose things where we could feel more comfortability.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/6567640" style="background-image: url('/wiki/shared/avatars/Realmaas.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-loved" href="https://osu.ppy.sh/users/6567640">Realmaas</a></p>
+
+After this year's win, motivation seems to be at an all-time high in the Argentinian community. What can we expect from your team in the future?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/8172283" style="background-image: url('/wiki/shared/avatars/-Story-.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/8172283">-Story-</a></p>
+
+I'm very happy with everyone who congratulated us, and I'm not just talking about Argentina when I say that we truly received a LOT of support. It's undoubtedly a great memory for us, and it means a lot, even though there's some support and also hate but we choose to focus on the effort it took to get here, day after day.
+
+None of this would have been achieved without the motivation we had as a team, as a family, as a friendship. We went through a lot but I would like to thank the friends i have as colleagues. My best friends are [Zeus](https://osu.ppy.sh/users/8329151), [Alex](https://osu.ppy.sh/users/17738253), [VelperK](https://osu.ppy.sh/users/348467), [Naarkz](https://osu.ppy.sh/users/19490380), [Gonza](https://osu.ppy.sh/users/17309439) and I want to say thanks for everything to them and for the supporters that were there every single match and day to pray for our win ❤️ 
```

Comment:
```suggestion
None of this would have been achieved without the motivation we had as a team, as a family, as a friendship. We went through a lot but I would like to thank the friends i have as colleagues. My best friends are [Zeus](https://osu.ppy.sh/users/8329151), [Alex](https://osu.ppy.sh/users/17738253), [VelperK](https://osu.ppy.sh/users/348467), [Naarkz](https://osu.ppy.sh/users/19490380), [Gonza](https://osu.ppy.sh/users/17309439) and I want to say thanks for everything to them and for the supporters that were there every single match and day to pray for our win ❤️
```